docs/packaging-applications/package-repositories/guides/container-registries/oracle-cloud-infrastructure-registry.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Oracle Cloud Infrastructure Container Registry  
+description: How to add an Oracle Cloud Infrastructure Container Registry as an Octopus feed 
+position: 90
+---
+
+Oracle Cloud Infrastructure (OCI) provides a container registry that can be used as an external feed for Octopus Deploy.
+
+## Create an OCI container registry
+Once you've logged into OCI, search for `Container Registry` and select the **Container Registry** link located under `Services`.
+
+![](images/oracle-cloud-infrastructure-container-registry-search.png)
+
+Click on **Create repository**
+
+![](images/oracle-cloud-infrastructure-create-registry.png)
+
+Fill in the following:
+- Compartment - select which compartment to place the repository in
+- Repoistory name - give the repository a name
+- Access - select whether this will be an `Public` or `Private` repository
+
+Click **Create repository**
+
+Take note of which region you created the repository in, this will be needed to determine the correct URL for the repository.  Use [https://docs.oracle.com/en-us/iaas/Content/General/Concepts/regions.htm](https://docs.oracle.com/en-us/iaas/Content/General/Concepts/regions.htm) to translate the region name to the shortened code.  In this example, the repository was created in `US West (San Jose)` which has the code of `sjc`.
```

Comment:
```suggestion
Take note of which region you created the repository in as this will be needed to determine the correct URL for the repository.  Use the [Oracle region documentation](https://docs.oracle.com/en-us/iaas/Content/General/Concepts/regions.htm) to lookup the **Region Key**.  In this example, the repository was created in `US West (San Jose)` which has the region key of `SJC`.
```